### PR TITLE
Updating charities sf

### PR DESCRIPTION
**Pull request by StephanieFriend**
### Type of change made:
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling - no new features
### Why is this change required? What problem does it solve?
Add two new post_parsers in the item controller for the added columns

### CODE REVIEW

File: application/controllers/items.py [hunk 1]
```diff
@@ -87,6 +89,20 @@
...
+item_post_parser.add_argument(
+    'charity_url',
+    type=str,
+    required=True,
+    location=['json'],
+    help='charity_url parameter is required'
+)
+item_post_parser.add_argument(
+    'charity_score',
+    type=str,
```
Comment (kmcgrevey) [inline application/controllers/items.py]: May be a conflict -- `charity_score` here is a string and and integer in the model

File: application/models/item.py [hunk 2]
```diff
@@ -12,6 +12,8 @@ class Item(db.Model):
     status = db.Column(db.String())
     category = db.Column(db.String())
     charity = db.Column(db.String())
+    charity_url = db.Column(db.String())
+    charity_score = db.Column(db.Integer())
```
Comment (kmcgrevey) [inline application/models/item.py:16]: Possible conflict -- `charity_score` is listed here as an Integer

File: application/controllers/items.py (hunk 1 above)
Comment (StephanieFriend) [reply application/controllers/items.py]: Thanks! new changes have been pushed up

File: application/models/item.py (hunk 2 above)
Comment (StephanieFriend) [reply application/models/item.py:16]: Pushed new changes up - keeping charity_score as an int

**Review by kmcgrevey — APPROVED** [review]

**Pull request merged**